src/snowflake/cli/plugins/snowpark/common.py

```diff
@@ -197,3 +202,15 @@ def format_arg(arg: Argument):
         schema=udf_sproc.schema_name,
     )
     return f"{name}({arguments})"
+
+
+def _get_import_from_resource(resource_json: dict) -> List[str] | None:
+    pattern = re.compile(r"@\w+\.\w+\.([^,]+)")
+    if "imports" in resource_json.keys():
+        return [
+            imp.lower()
+            for imp in pattern.findall(resource_json["imports"])
+            if "app.zip" not in imp
```

Comment:
Why app.zip is hardcoded? Add tests for different source name

Reply:
Changed